README.md

```diff
@@ -13,3 +13,4 @@ Projects from students of NLP Course
 |Text Summarization Task in Russian|The problem which I am going to solve is summarization task in Russian. Nowadays, we have a lot of information and it is important to extract the main idea from a text, in my case the model will help people to generate headlines for news articles.|@alexvishnevskiy| https://github.com/alexvishnevskiy/Huawei-project |
 |Generation of news headlines|Summarization task in Russian for news data set|@germanjke, @kotyukov|https://github.com/kotyukov/huaweiNLP https://github.com/germanjke/huaweiNLP |
 |Russian aspect-based sentiment analysis|BERT-based techniques to identify the sentiment of the selected entity in the text. For example, "In general I like the car but I hate it's <color>". The sentiment of the "color" is negative.The most relevant dataset is https://github.com/songyouwei/ABSA-PyTorch/tree/master/datasets/semeval14 |@preduct0r| https://github.com/preduct0r/huawei|
+|Jigsaw Multilingual Toxic Comment Classification|https://www.kaggle.com/c/jigsaw-multilingual-toxic-comment-classification|LeonidMorozov, Mteterin|https://github.com/LeonidMorozov/jigsaw_toxic_classification|
```

Comment:
please make more detailed description

Reply:
@madrugado ,
This is kaggle competition. Do you want me to copy/paste description from kaggle? I'm not follow, sorry.

Reply:
I want you to describe it in one-two sentences as the other projects do.